readme.md

```diff
@@ -146,6 +146,16 @@ Default: `','`
 
 The character used to separate array elements when using `{arrayFormat: 'separator'}`.
 
+```js
+const queryString = require('query-string');
+
+queryString.parse('foo=bar|baz', {
+	arrayFormat: 'separator',
+	arrayFormatSeparator: '|'
+});
+//=> {foo: ['bar', 'baz']}
+```
```

Comment:
Can you add these examples to https://github.com/sindresorhus/query-string/blob/master/index.d.ts too?